hyrax/app/models/nims_roles.rb

```diff
@@ -1,7 +1,7 @@
 module NIMSRoles
   def authenticated_nims_researcher?
     unless defined?(@authenticated_nims_researcher)
-      if !guest? && employee_type_code.present? && employee_type_code =~ /^(A|G|L|Q|R|S)/i
+      if !guest? && employee_type_code.present? && employee_type_code =~ /^1[1-3]$/i
```

Comment:
This would not be satisfied for any user in the event where we keep authenticating with LDAP.